src/boot/lib/mexpr.ml

```diff
@@ -1484,6 +1484,8 @@ let rec eval (env : (Symb.t * tm) list) (t : tm) =
 let rec eval_toplevel (env : (Symb.t * tm) list) = function
   | TmLet (_, _, s, _ty, t1, t2) ->
       eval_toplevel ((s, eval env t1) :: env) t2
+  | TmType (_, _, _, _, t1) ->
```

Comment:
Ah, I missed adding this when I revised the boot types. A suggestion: replace the catch all case for this match with an explicit list of remaining terms. Then, we'll get compiler errors when adding new terms such as `TmType`.

Reply:
Fixed!